src/consumer.cpp

```diff
@@ -26,7 +26,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
-
+#include <syslog.h>
```

Comment:
This header doesn't exist on Windows. I'm not sure how rdkafka does this so it works on all platforms but this should do the same.